spec/lib/driver_commission_history_response_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'spec_helper'
+require 'gman_client'
+require 'vcr'
+require 'httparty'
+require 'webmock'
+require 'support/vcr'
+
+RSpec.describe GmanClient do
+  describe '.driver_commission_histories' do
+    VCR.use_cassette('driver_commission_histories') do
+      response = GmanClient.driver_commission_histories
```

Comment:
I localized teh VCR inside the let statement.
``Ruby
    let(:response) do
      VCR.use_cassette('driver_commission_histories') do
        GmanClient.driver_commission_histories
      end
    end
```
Works better and faster.